src/viam/sdk/spatialmath/orientation.cpp

```diff
@@ -77,8 +77,9 @@ OrientationConfig OrientationConfig::from_proto(proto::Orientation proto) {
             quat.y = 0;
             quat.z = 0;
```

Comment:
Should there be a break here? It seems strange to build an object just to throw an exception

Reply:
🤦 yes there absolutely should be. thanks for catching that!